src/P0wnedPassword/Request/Client.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the RollerworksPasswordStrengthValidator package.
+ *
+ * (c) Sebastiaan Stok <s.stok@rollerscapes.net>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Rollerworks\Component\PasswordStrength\P0wnedPassword\Request;
+
+use GuzzleHttp\Psr7\Request;
```

Comment:
Is it possible to use the Request object of `php-http/httplug`? This would remove the requirement to use Guzzle, which is exactly why we have PSR-7 😄

Reply:
I wanted to but php-http/httplug has no Request object. So the option is to create our own PSR-7 compatible Request object or to use a PSR7 implementing library. 

Also this doesn't pull in guzzle, it only pulls in a PSR-7 implementation. See how small it is here: <https://github.com/guzzle/psr7/tree/master/src>

Reply:
OK 👍 keep it.